cmd/skaffold/app/cmd/delete.go

```diff
@@ -48,6 +48,7 @@ func delete(out io.Writer) error {
 	if err != nil {
 		return errors.Wrap(err, "creating runner")
 	}
+	runner.RPCServerShutdown()
```

Comment:
this should be `defer`red? and should probably return an error and log it if there is an issue?